components/eamxx/src/share/io/scorpio_output.cpp

```diff
@@ -202,6 +202,18 @@ AtmosphereOutput (const ekat::Comm& comm, const ekat::ParameterList& params,
   // Register any diagnostics needed by this output stream
   set_diagnostics();
 
+  // Helper lambda, to copy io string attributes. This will be used if any
+  // remapper is created, to ensure atts set by atm_procs are not lost
+  const std::string io_string_atts_key ="io: string attributes";
```

Comment:
any reason this is not defined inside the lambda?  I don't think you use it anywhere else in the constructor.